modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java

```diff
@@ -157,14 +157,6 @@ public static String camelize(final String inputWord, boolean lowercaseFirstLett
                 m = camelizeSlashPattern.matcher(word);
             }
 
-            // Uppercase the class name.
```

Comment:
Instead of removing this code block, have you considered using the boolean flag `lowercaseFirstLetter` in this function?

Reply:
Yes but we want the first letter as uppercase.
If you really want to keep this behavior of StringUtils, maybe I can try to rework the getter/setter generation. WDYT?

Reply:
I wouldn't suggest we change this method as it's impacting too many generators.

What about creating another method `camelizeWithDollarSign` to support this particular use case?

Reply:
I look at your idea, but instead of creating a new specific method for the java use case with a dollar in getter names, I found another solution: having the same name for the property.
The main issue was that we use `lowercaseFirstLetter` but what is really wanted (for me at least) is lower case the first char.
So I make the choice possible in the `camelize` method by replacing the boolean with an option to select if we want
- UPPERCASE_FIRST_CHAR (= current false)
- LOWERCASE_FIRST_LETTER (= current true)
- LOWERCASE_FIRST_CHAR (= new behavior)

I know that impact a lot of code, but I think it's cleaner than adding another boolean or another method with nearly the same name.
I think the change is ok because even without the dollar, we have this change:
`Model200Response _200response = (Model200Response) o;`
to
`Model200Response _200Response = (Model200Response) o;`
And now the name case matches the className case.

Reply:
Please PM me via Slack on coming Monday to discuss this further.